Documentation/helm-values.rst

```diff
@@ -90,6 +90,18 @@
      - Seconds after which the completed job pod will be deleted
      - int
      - ``1800``
+   * - cgroup
+     - Configure cgroup related configuration
+     - object
+     - ``{"autoMount":{"enabled":true},"root":"/run/cilium/cgroupv2"}``
+   * - cgroup.autoMount.enabled
+     - Enable auto mount of cgroup v2 filesystem
+     - bool
+     - ``true``
+   * - cgroup.root
+     - Configure cgroup root where cgroup v2 filesystem is mounted
```

Comment:
The above line seems a bit confusing to me. How about: "Location where the cgroup v2 filesystem is mounted inside the cilium agent container"?